harness/determined/experimental/_core_context_v2.py

```diff
@@ -0,0 +1,174 @@
+import logging
+import pathlib
+from typing import Any, Dict, Optional, Union
+
+import appdirs
+
+import determined as det
+from determined import core, experimental, tensorboard
+from determined.common import api, storage, util
+from determined.common.api import certs
+
+logger = logging.getLogger("determined.experimental.core_v2")
+
+
+def _default_storage_manager() -> storage.SharedFSStorageManager:
+    base_path = pathlib.Path(appdirs.user_data_dir("determined")) / "checkpoints"
+    if not base_path.exists():
+        base_path.mkdir(parents=True)
+
+    logger.info(f"no storage_manager provided; storing checkpoints in {str(base_path)}")
+    storage_manager = storage.SharedFSStorageManager(str(base_path))
+    return storage_manager
+
+
+def init(
```

Comment:
is this dead code?

Or... where is this called?  It seems to conflict with core_v2.py's `init()` call in name and purpose.

Reply:
it's used to build the `Context` by `core_v2.py`. it is called `init()` because it's the Core API v1's `init` with extra features... guess it should be called `_init`. it should be possible to port Core API v1's `init` to this method.

Reply:
Maybe call it `_make_v2_context()` or something?  It is confusing to me to see so many different things called `*init()`.

But at least, the import and usage pattern you have for it makes more sense to me now, even if you leave it called `_init()`.  So consider this an "optional" comment.